### PR TITLE
added in sort function + added in rendering of owner responses

### PR DESCRIPTION
**Pull request by djian001**
woohooo got my sort to work.

### CODE REVIEW

File: client/src/app.jsx [hunk 1]
```diff
@@ -2,48 +2,67 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import $ from 'jquery';
 import ReviewList from './ReviewList.jsx';
+import helpers from './helpers.js';
```
Comment (kimberlyjk) [inline client/src/app.jsx:5]: smart!

File: client/src/app.jsx [hunk 2]
```diff
@@ -2,48 +2,67 @@ import React from 'react';
...
-          <div className='review'>
-            <ReviewList list={this.state.list}/>
-          </div>
+    //console.log('e', option)
 
-       )
+    let sortList = helpers.sortReviews(this.state.list, option);
+    //console.log('sorted', sortList);
 
+    this.setState({
```
Comment (kimberlyjk) [inline client/src/app.jsx:49]: nice use of the helpers and setState to keep this code "clean"

File: client/src/reviewList.jsx [hunk 3]
```diff
@@ -41,7 +42,9 @@ function ReviewList(props) {
                 <i className="far fa-thumbs-up"></i> Helpful {item.helpfulness}
               </button>
               <div className="owner">
-                owner responses here
+                <div className="owner-date">{moment(item.ownerResponse.responseDate).format('MMMM Do YYYY')}</div>
```
Comment (kimberlyjk) [inline client/src/reviewList.jsx:45]: awesome! way to get the owner responses to be added. 

I did notice that on Michael's review there is an 'Invalid date' shown, you might want to add logic in the case that the owner hasn't left a response.

**Review by kimberlyjk — APPROVED** [review]
lots of good work here Denise! I think there was one thing I noticed and it was just to add logic to the code in the case where an owner has not responded to a post. 

keep it up!

File: client/src/reviewList.jsx (hunk 3 above)
Comment (djian001) [reply client/src/reviewList.jsx:45]: Yeah I haven't gotten to this yet but there's a couple of logic stuff I need to implement

**Pull request merged**